### PR TITLE
Fix error related with ‘getDefaultProps’

### PR DESCRIPTION
**Pull request by ifndefdeadmau5**
Activity were died because of previous pull request and now fixed.

### CODE REVIEW

File: src/components/Activity/DefaultTabBar.js [hunk 1]
```diff
@@ -1,33 +1,41 @@
...
+      backgroundColor: null,
+    };
+  }
+
   renderTab(name, page, isTabActive, onPressHandler) {
-    const { activeTextColor, inactiveTextColor, textStyle, } = this.props;
+    const { activeTextColor, inactiveTextColor, textStyle, } = DefaultTabBar.defaultProps;
     const textColor = isTabActive ? activeTextColor : inactiveTextColor;
-    const fontWeight = isTabActive ? 'bold' : 'normal';
+    const fontWeight = isTabActive ? 'bold' : 'bold';
```
Comment (heartjigi) [inline src/components/Activity/DefaultTabBar.js:23]: const fontWeight = 'bold' right?

Comment (SarahhhC) [inline src/components/Activity/DefaultTabBar.js]: Always `bold`?

File: src/components/Activity/DefaultTabBar.js [hunk 2]
```diff
@@ -47,29 +55,28 @@ class DefaultTabBar extends Component {
...
-      bottom: 0,
+      bottom: 0.5,
       zIndex: -1,
     };
 
     const left = this.props.scrollValue.interpolate({
-      inputRange: [0, 1, ],
-      outputRange: [this.props.leftOffset,  (containerWidth / 2) - this.props.rightOffset, ],
+      inputRange: [0, 1],
+      outputRange: [this.props.leftOffset,  (containerWidth / 2) - this.props.rightOffset],
```
Comment (heartjigi) [inline src/components/Activity/DefaultTabBar.js]: 2 spaces before (containerWidth / 2)

**Comment by SarahhhC** [comment]
Finished review! After adjust some comments merge it. LGTM

[Approved with PullApprove](https://pullapprove.com/sv-bootcamp/bridgeme-client/pull-request/151/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=sv-bootcamp/bridgeme-client)

File: src/components/Activity/DefaultTabBar.js [hunk 3]
```diff
@@ -1,33 +1,41 @@
...
+import {
+  Animated,
   Dimensions,
   StyleSheet,
-  Text,
   View,
-} = ReactNative;
-const Button = require('./Button');
+
} from 'react-native';
```
Comment (MechanicKim) [inline src/components/Activity/DefaultTabBar.js]: 7:1 Irregular whitespace not allowed(no-irregular-whitespace)
7:1 Expected linebreaks to be 'LF' but found 'CRLF'(linebreak-style)

These errors are difficult to find but I think the errors may have a potential problem.

File: src/components/Activity/DefaultTabBar.js [hunk 4]
```diff
@@ -1,33 +1,41 @@
...
+    return {
+      activeTextColor: '#2e3031',
+      inactiveTextColor: '#adadad',
+      backgroundColor: null,
+    };
+  }
+
   renderTab(name, page, isTabActive, onPressHandler) {
-    const { activeTextColor, inactiveTextColor, textStyle, } = this.props;
+    const { activeTextColor, inactiveTextColor, textStyle, } = DefaultTabBar.defaultProps;
```
Comment (MechanicKim) [inline src/components/Activity/DefaultTabBar.js]: 21:58 Unexpected trailing comma(comma-dangle)

File: src/components/Activity/DefaultTabBar.js [hunk 5]
```diff
@@ -1,33 +1,41 @@
...
     const textColor = isTabActive ? activeTextColor : inactiveTextColor;
-    const fontWeight = isTabActive ? 'bold' : 'normal';
+    const fontWeight = isTabActive ? 'bold' : 'bold';
 
-    return (<Button
+    return <Button
```
Comment (MechanicKim) [inline src/components/Activity/DefaultTabBar.js]: 25:12 Missing parentheses around multilines JSX(react/jsx-wrap-multilines)

File: src/components/Activity/DefaultTabBar.js [hunk 6]
```diff
@@ -1,33 +1,41 @@
...
       style={{ flex: 1, }}
```
Comment (MechanicKim) [inline src/components/Activity/DefaultTabBar.js]: 26:23 Unexpected trailing comma(comma-dangle)

File: src/components/Activity/DefaultTabBar.js [hunk 7]
```diff
@@ -1,33 +1,41 @@
...
       key={name}
       accessible={true}
```
Comment (MechanicKim) [inline src/components/Activity/DefaultTabBar.js:28]: 28:7 Value must be omitted for boolean attributes(react/jsx-boolean-value)

File: src/components/Activity/DefaultTabBar.js [hunk 8]
```diff
@@ -1,33 +1,41 @@
...
       accessibilityLabel={name}
       accessibilityTraits='button'
```
Comment (MechanicKim) [inline src/components/Activity/DefaultTabBar.js]: 31:27 Unexpected usage of singlequote(jsx-quotes)

File: src/components/Activity/DefaultTabBar.js [hunk 9]
```diff
@@ -1,33 +1,41 @@
...
       onPress={() => onPressHandler(page)}
     >
-      <View style={[styles.tab, this.props.tabStyle,]}>
-        <Text style={[{ color: textColor, fontWeight, }, textStyle,]}>
+      <View style={[styles.tab]}>
+        <Text style={[{ color: textColor, fontWeight, }, textStyle]}>
```
Comment (MechanicKim) [inline src/components/Activity/DefaultTabBar.js]: 34:53 Unexpected trailing comma(comma-dangle)

File: src/components/Activity/DefaultTabBar.js [hunk 10]
```diff
@@ -47,29 +55,28 @@ class DefaultTabBar extends Component {
...
 
     return (
       <View>
-        <View
-          style={[
-            styles.tabs,
-            { backgroundColor: this.props.backgroundColor, },
-            this.props.style,
-            ]}>
+        <View style={[
```
Comment (MechanicKim) [inline src/components/Activity/DefaultTabBar.js]: 69:15 Property should be placed on a new line(react/jsx-first-prop-new-line)

File: src/components/Activity/DefaultTabBar.js [hunk 11]
```diff
@@ -47,29 +55,28 @@ class DefaultTabBar extends Component {
...
+          styles.tabs,
+          { backgroundColor: this.props.backgroundColor, },
```
Comment (MechanicKim) [inline src/components/Activity/DefaultTabBar.js]: 71:56 Unexpected trailing comma(comma-dangle)

File: src/components/Activity/DefaultTabBar.js [hunk 12]
```diff
@@ -47,29 +55,28 @@ class DefaultTabBar extends Component {
...
+          this.props.style,
+          ]}>
```
Comment (MechanicKim) [inline src/components/Activity/DefaultTabBar.js:74]: 73:11 Expected indentation of 8 spaces but found 10
73:13 The closing bracket must be aligned with the line containing the opening tag (expected column 9 on the next line)

File: src/components/Activity/DefaultTabBar.js [hunk 13]
```diff
@@ -47,29 +55,28 @@ class DefaultTabBar extends Component {
...
           {this.props.tabs.map((name, page) => {
             const isTabActive = this.props.activeTab === page;
             const renderTab = this.props.renderTab || this.renderTab;
             return renderTab(name, page, isTabActive, this.props.goToPage);
           })}
-          <Animated.View style={[tabUnderlineStyle, { left, }, this.props.underlineStyle, ]} />
+          <Animated.View style={[tabUnderlineStyle, { left, }, this.props.underlineStyle]} />
```
Comment (MechanicKim) [inline src/components/Activity/DefaultTabBar.js:80]: 79:59 Unexpected trailing comma(comma-dangle)

File: src/components/Activity/DefaultTabBar.js (hunk 4 above)
Comment (ifndefdeadmau5) [reply src/components/Activity/DefaultTabBar.js]: ESLint says only IE8 and below browser occurs error with this, So isn't it good to use trailing comma in React-Native?

File: src/components/Activity/DefaultTabBar.js (hunk 1 above)
Comment (ifndefdeadmau5) [reply src/components/Activity/DefaultTabBar.js:23]: Oh you're right

File: src/components/Activity/DefaultTabBar.js (hunk 2 above)
Comment (ifndefdeadmau5) [reply src/components/Activity/DefaultTabBar.js]: Good

**Pull request merged**